chore(docs): General 0.6 doc improvements (#366)
grain-lang/grain-lang.org: src/guide/basics.md

true || false // true
!true // false
```

### Block expressions

> Block expressions allow you to group several expressions together to be executed one after another: to write a block expression simply surround several expressions in curly braces. The block expression as a whole will evaluate to its last expression's value. For example:

Reviewer: ```suggestion
Block expressions allow you to group several expressions together to be executed one after another: to write a block expression surround several expressions with curly braces. The block expression as a whole will evaluate to its last expression's value. For example:
```
